homeassistant/components/rainbird/quality_scale.yaml

```diff
@@ -43,16 +43,22 @@ rules:
       needs to be moved to `runtime_data`.
 
   # Silver
-  log-when-unavailable: todo
-  config-entry-unloading: todo
+  log-when-unavailable: done
+  config-entry-unloading: done
   reauthentication-flow: done
-  action-exceptions: todo
-  docs-installation-parameters: todo
-  integration-owner: todo
-  parallel-updates: todo
-  test-coverage: todo
-  docs-configuration-parameters: todo
-  entity-unavailable: todo
+  action-exceptions: done
+  docs-installation-parameters:
+    status: todo
+    comment: The documentation does not mention installation parameters
+  integration-owner: done
+  parallel-updates:
+    status: todo
+    comment: The integration does not explicitly set a number of parallel updates.
```

Comment:
This appears to be an even lower level problem.  Even when then only client is a single update coordinator only sending one request per minute, we still see issues with the number of open tcp connections. That is, it would send a single request, then 60 seconds later get a "device busy" error because of a previous tcp connection that was open.

I do think setting `PARALLEL_UPDATES` here can still make sense, so left as a TODO to do that.  Using `asyncio.Lock` in the library wouldn't necessarily hurt either, but not sure it entirely solves it.